app/src/main/kotlin/com/flxrs/dankchat/chat/ChatAdapter.kt

```diff
@@ -635,6 +643,19 @@ class ChatAdapter(
         }
     }
 
+    /** make common styles for timestamp part of the message
+     * @param context: used for retreiving style resources (just pass view's context)
+     */
+    private fun makeTimestampSpanStyles(context: Context): Array<CharacterStyle> {
```

Comment:
I would prefer an inline extension function similar to the existing ones, like the ` bold` one. Returning an array and having to spread it for the `inSpans()` is just ugly xD